apps/site/assets/ts/tnm/components/RouteCard.tsx

```diff
@@ -44,12 +47,23 @@ const RouteCard = ({
     return null;
   }
 
+  const hasAlert = route.route.alerts.length > 0;
```

Comment:
Would it make sense to double check that `route.route.alerts` exists? You could use lodash's `get`:
`get(route, "route.alerts", <some default value>)`